server/events/pending_plan_finder_test.go

```diff
@@ -282,3 +282,19 @@ func runCmd(t *testing.T, dir string, name string, args ...string) string {
 	Assert(t, err == nil, "err running %q: %s", strings.Join(append([]string{name}, args...), " "), cpOut)
 	return string(cpOut)
 }
+
+func runCmdErrCode(t *testing.T, dir string, errCode int, name string, args ...string) string {
```

Comment:
you can move test utility into `testing` directory.

Reply:
You're saying to move this function into https://github.com/runatlantis/atlantis/tree/781d9ac76c2e99c6a5755a426edd2655211a2eeb/testing ?

Reply:
yes. test utility function is used like this.  https://github.com/runatlantis/atlantis/blob/781d9ac76c2e99c6a5755a426edd2655211a2eeb/server/events/models/models_test.go#L22

If this is useful for other test cases, it might be better to move it into testing dir.